phpstan.neon

```diff
@@ -0,0 +1,5 @@
+parameters:
```

Comment:
Maybe name this file `phpstan.neon.dist`?

Reply:
Done!